applications/DEM_application/custom_elements/cluster3D.cpp

```diff
@@ -329,7 +251,6 @@ namespace Kratos {
         }        
     }
     
-    
     void Cluster3D::UpdatePositionOfSpheres() {
         
```

Comment:
This function could be overwriting 'UpdatePositionOfNodes', in the RigidBodyElement3D